src/figuresClasses.ts

```diff
@@ -1,19 +1,101 @@
 export interface Figure {
-
+  shape: string
+  color: string
+  getArea(): number | never
+  sides?: number[]
+  radius?: number
```

Comment:
In Figure interface you should specify only 3 main properties, sides and radius should be created for specific figures (sides only for Triangles, radius only for Circles)